docs/content/configuration/filter.md

```diff
@@ -68,3 +68,19 @@ D enterprise.github.com/company
 </pre>
 
 In the above example, any module not in the rules will be excluded.  All modules from `enterprise.github.com/company` are fetched directly from the source.  The `github.com/gomods/athens` module will be stored in the proxy storage, but only for version `v0.4.1` and any patch versions under `v0.1` and `v0.2` minor versions.
+
+### Versions Filter Modifiers
+
+Athens provides advanced filter modifiers to cover cases such as api compatibility or when a given dependency changes its license from a given versions. The modifiers are intended to be used in the pattern list of the filter file.
+
+<pre>
+-
+# external dependency approved list
++ github.com/gomods/athens <v1.2.3
+</pre>
+
+The currently supported modifiers are 
+
+- ~v1.2.3: enables 1.2.x  which are at least 1.2.3
+- ^v1.2.3: enables 1.x.x which are at least 1.2.3
+- <v1.2.3: enables everything lower than 1.2.3 includes 1.2.2 and 0.58.9 as well
```

Comment:
I have some wording suggestions to make this more human readable and more formally specific

* `~1.2.3` will enable all patch versions from 1.2.3 and above (e.g. 1.2.3, 1.2.4, 1.2.5)
    * Formally, `1.2.x` where `x >= 3`
* `^1.2.3` will enable all patch and minor versions from 1.2.3 and above (e.g. 1.2.4, 1.3.0 and 1.4.5)      
    * Formally, `1.x.y` where `x >= 2` and `y >= 3`
* `<1.2.3` will enable all versions lower than 1.2.3 (e.g. 1.2.2, 1.0.0 and 0.58.9)
    * Formally, `x.y.z` where `x <= 1`, `y < = 2` and `z < 3`

Reply:
Done!